crates/biome_js_formatter/src/utils/format_modifiers.rs

```diff
@@ -26,6 +26,10 @@ where
         let modifiers = sort_modifiers_by_precedence(&self.list);
         let should_expand = should_expand_decorators(&self.list);
 
+        if self.list.is_empty() {
+            return Ok(());
+        }
```

Comment:
Returning early here is important, because otherwise this node returns a group that _always_ has a `soft_line_break_or_space`, which causes `may_directly_break` to return true. An example case this affects is:

```typescript
class Test {
  prop1 = // comment
    true;
}
```

Without this early escape, the IR gets written as:
```
group(expand: propagated, [
  group(expand: true, [soft_line_break_or_space]),
  " prop1",
  line_suffix([" // comment"]),
  expand_parent
]),
" =  true"
```
and that group with just the soft line is this modifier list. With this early return, that group no longer exists, meaning the outer group won't contain any potential breaks, and the comments get formatted as expected by Prettier.

Reply:
This is a great comment, and a I think we should have it **inside** the code :) (maybe a variation of that)